src/components/UserAvatar.js

```diff
@@ -1,40 +1,45 @@
-import { Box, Link, Avatar, Typography } from '@mui/material';
+import { Box, Link, Avatar, Typography, Tooltip } from '@mui/material';
 
 function UserAvatar({ auth }) {
   if (!auth?.isAuthenticated) {
     return null;
   }
+
+  const userName = auth?.user?.profile.preferred_username[0] || null;
+
   return (
-    <Box>
-      <Link
-        href="https://dev-k8s.treetracker.org/keycloak/realms/treetracker/account/#/personal-info"
-        target="_blank"
-        rel="noopener noreferrer"
-        sx={{
-          color: ({ palette }) => palette.text.primary,
-        }}
-      >
-        <Avatar
+    <Tooltip title={userName} arrow>
```

Comment:
Currently the arrow tooltip will show up behind the header/banner of the website. This is likely due to the header having a z-index of '9999'.

![Screenshot 2024-01-30 191156](https://github.com/Greenstand/treetracker-web-map-client/assets/116132122/d14f45be-67e0-40ea-b98a-531ea95468eb)

 I am hesitant to suggest changing the header zindex to anything lower in the event it ruins the format in some place I'm unaware of. I am open to suggestions or ideas. @dadiorchen ?